optical/converter/yolo.py

```diff
@@ -1,17 +1,136 @@
 """
-__author__: HashTagML
+__author__:HashTagML
 license: MIT
 Created: Wednesday, 31st March 2021
 """
 
-from .base import FormatSpec
```

Comment:
Please follow this:
there is a pattern of organizing your imports. it goes like this:

=========

import packages that comes with base python e.g., os, pathlib, json etc.

import third-party packages e.g. numpy, pandas, tqdm etc.

import modules from the projects e.g. the relative imports

=========
There is a command in vscode in the command palette called "organize imports" which does that automatically for you.